designer/client/src/components/toolbarComponents/toolbarWrapper/ToolbarWrapper.tsx

```diff
@@ -7,8 +7,9 @@ import { variables } from "../../../stylesheets/variables";
 import { useDragHandler } from "../DragHandle";
 import { CollapsiblePanelContent, Panel, PanelHeader } from "../Panel";
 import { IconWrapper, StyledCloseIcon, StyledCollapseIcon, Title } from "./ToolbarStyled";
+import { useTheme } from "@mui/material";
 
-const { panelBackground, sidebarWidth } = variables;
+const { sidebarWidth } = variables;
```

Comment:
Are we using variables?

Reply:
Yes, I left variables in cases when the value is a real variable, for example in the case of element size, where there is no use in moving it to the theme because this value will be always the same, no matter which theme